network/model.go

```diff
@@ -11,3 +11,121 @@ type CreateInteractionRequest struct {
 type CreateGameRequest struct {
 	PlayerIDs []model.PlayerID `json:"playerIDs"`
 }
+
+// TODO figure out the minimum info the client will need
+type GetGameResponse struct {
+	ID              model.GameID                         `json:"id"`
+	Players         []Player                             `json:"players"`
+	PlayerColors    map[model.PlayerID]model.PlayerColor `json:"player_colors,omitempty"`
+	CurrentScores   map[model.PlayerColor]int            `json:"current_scores"`
+	LagScores       map[model.PlayerColor]int            `json:"lag_scores"`
+	Phase           model.Phase                          `json:"phase"`
+	BlockingPlayers map[model.PlayerID]model.Blocker     `json:"blocking_players,omitempty"`
+	CurrentDealer   model.PlayerID                       `json:"current_dealer"`
+	Hands           map[model.PlayerID][]model.Card      `json:"hands,omitempty"`
+	Crib            []model.Card                         `json:"crib,omitempty"`
+	CutCard         model.Card                           `json:"cut_card"`
+	PeggedCards     []model.PeggedCard                   `json:"pegged_cards,omitempty"`
+}
+
+func NewGetGameResponse(g model.Game) GetGameResponse {
+	return GetGameResponse{
+		ID:              g.ID,
+		Players:         newPlayersFromPlayerModels(g.Players),
+		PlayerColors:    g.PlayerColors,
+		CurrentScores:   g.CurrentScores,
+		LagScores:       g.LagScores,
+		Phase:           g.Phase,
+		BlockingPlayers: g.BlockingPlayers,
+		CurrentDealer:   g.CurrentDealer,
+		Hands:           g.Hands,
+		Crib:            g.Crib,
+		CutCard:         g.CutCard,
+		PeggedCards:     g.PeggedCards,
+	}
+}
```

Comment:
I don't love how copypasta this is between `GetGameResponse` and `CreateGameResponse`. I know there's a _chance_ that they diverge in the future, but is there a way to be more DRY?